dream_layer_backend/txt2img_server.py

```
@@ -1,22 +1,13 @@
-import copy
-import random
 from dream_layer import get_directories
-from dream_layer_backend_utils.update_custom_workflow import update_custom_workflow, validate_custom_workflow, find_save_node
-from dream_layer_backend_utils.api_key_injector import inject_api_keys_into_workflow
-from dream_layer_backend_utils.workflow_loader import load_workflow, analyze_workflow
-from dream_layer_backend_utils.update_custom_workflow import override_workflow
 from dream_layer_backend_utils import interrupt_workflow
-from shared_utils import wait_for_image, send_to_comfyui, serve_image
+from shared_utils import  send_to_comfyui
 from dream_layer_backend_utils.fetch_advanced_models import get_controlnet_models
 from PIL import Image, ImageDraw
```

Ruff (0.11.9)

`PIL.Image` imported but unused

Remove unused import

(F401)